src/main/kotlin/_3_racing_car/service/dto/RacingOutputDto.kt

```diff
@@ -0,0 +1,13 @@
+package _3_racing_car.service.dto
+
+data class RacingOutputDto(val racers: Map<String, Int>) {
+
+    var maxLocation: Int?
+        private set
+
+    init {
+        maxLocation = racers.maxByOrNull { it.value }?.value
+    }
```

Comment:
```suggestion
    val maxLocation: Int = racers.maxOfOrNull { it.value } ?: 0
```

코틀린에서는 많은 컬렉션 함수가 제공되니, 최대한 값들은 불변으로 유지해봐요!